Header header
geometry_msgs/Pose2D pose
uint8 type
uint32 id

uint8 UNKNOWN = 0
uint8 SMALL = 1
uint8 LARGE = 2
